### ISSUE TITLE
Windows 10 Messenger App Not Playing Voice Messages

### ISSUE THREAD

**Issue by 19darion91** [Sytten/Facebook-Messenger-Desktop#44]
So yeah... The app doesn't load voice messages - a.k.a it just keeps loading them, but it doesn't play them at all... Just the loading symbol is there. This is an issue I encountered on Windows 10 Pro 64bit. If anyone uses it on other system please check if the issue is there and leave a comment with an Yes or No answer and your system. Cheers!

**Comment by Sytten** [comment]
Yep known issue, should be resolved in the next official release on the original repo. Working on it.

**Comment by Sytten** [comment]
Should be resolved, but I won't close the issue since its a manual process that still needs to be scripted.
